src/databricks/labs/blueprint/wheels.py

```diff
@@ -240,20 +260,20 @@ def _now_iso():
     def __enter__(self) -> "WheelsV2":
         """Builds the wheel and returns the instance. Use it as a context manager."""
         self._tmp_dir = tempfile.TemporaryDirectory()
-        self._local_wheel = self._build_wheel(self._tmp_dir.name, verbose=self._verbose)
+        self._local_wheel = next(self._build_wheel(self._tmp_dir.name, verbose=self._verbose, no_deps=True))
```

Comment:
The usage of next() here should be tested, if no_deps=False then the output of _build_wheel should be all the wheels paths and not only the main one.